docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -438,6 +439,10 @@ Here's a simple lookup plugin implementation --- this lookup returns the content
 
           return ret
 
+.. note::
+
+  Use `plugin_obj.loader` to access the DataLoader APIs instead of using `plugin_obj._loader`.
+  As per architecture, not all plugins have access to `plugin_obj.loader`.
```

Comment:
I feel like something is missing here and this needs some clarification. "As per Ansible's architecture" perhaps but is it even more specific to plugins?

Reply:
I think this is a misunderstanding of my comments in the original ticket. This note is incorrect.